hardware_interface/src/mock_components/generic_system.cpp

```diff
@@ -29,6 +30,18 @@
 
 namespace mock_components
 {
+double parseDouble(const std::string & text)
```

Comment:
We are using "snake_case" for methods:

```suggestion
double parse_double(const std::string & text)
```